src/views/CardCreatorPage.vue

```diff
@@ -847,6 +829,9 @@ export default {
     }
   },
   methods: {
+    isEmpty (a) {
```

Comment:
I dont think this is necessary

Reply:
It is used in L554 
https://github.com/DecentralCardGame/frontend/pull/265/files#diff-048c994414ac950e6ee9aa6f7f6a1278da659f026c01c550027aaf0c71be70beR554

Before it was changed to !abilities which did not work properly

Reply:
I mean the wrapper might not be necessary